src/lightgbm/src/test/scala/VerifyLightGBMClassifier.scala

```diff
@@ -24,6 +25,11 @@ class VerifyLightGBMClassifier extends Benchmarks with EstimatorFuzzing[LightGBM
   verifyLearnerOnBinaryCsvFile("random.forest.train.csv",          "#Malignant", 1)
   verifyLearnerOnBinaryCsvFile("transfusion.csv",                  "Donated", 1)
 
+  verifySaveBooster(
+    fileName = "PimaIndian.csv",
+    labelColumnName = "Diabetes mellitus",
+    outputFileName = "target/model.txt")
```

Comment:
Unfortunately, This wont work to save into the target directory you need to use something like:

val targetDir = getClass.getResource("/")